examples/fedprox_example/client.py

```diff
@@ -53,6 +54,20 @@ def get_criterion(self, config: Config) -> _Loss:
         help="Seed for the random number generator",
         required=False,
     )
+    parser.add_argument(
+        "--pretrained_model_dir",
+        action="store",
+        type=str,
+        help="Path to the pretrained model",
+        required=False,
+    )
+    parser.add_argument(
+        "--weights_mapping_dir",
+        action="store",
+        type=str,
+        help="Path to the weights mapping file",
```

Comment:
That help text in inconsistent with the argument name. Either change the help text to say "files" or "directory" instead of a single file, change the argument name to `--weights_mapping_file`, or something else along those lines.

Reply:
Good catch, I will fix it.